.sbtopts

```diff
@@ -0,0 +1,4 @@
+-J-Xmx2048M
+-J-Xms512M
+-J-XX:MaxMetaspaceSize=512M
```

Comment:
Do we really need to limit metaspace?

Reply:
sbt default for max metaspace is 256M, I had to increase that limit to get akka compiling